packages/rulesets/src/spectral/functions/put-request-response-scheme.ts

```diff
@@ -0,0 +1,39 @@
+// Verifies if a PUT operation request and response schemas match.
+
+import lodash from "lodash";
+
```

Comment:
please avoid introducing new packages, as it will fails when using spectral cli to run the ruleset